### ISSUE TITLE
Mod won'

### ISSUE THREAD

**Issue by theLine** [zenyr/PocoHud3#52]

**Comment by djmattyg007** [comment]
You probably want this version: https://github.com/Kilandor/PocoHud3

**Comment by zenyr** [comment]
Yes, this version is loosely compatible with both versions but since the original hook is obsolete I think it is now reasonable to transform this main repo to adopt BLT structure.
